src/main/java/com/manager/voucher/domain/Item.java

```diff
@@ -0,0 +1,21 @@
+package com.manager.voucher.domain;
+
+public class Item {
+    private int price;
+
+    public Item(int price) {
+        this.price = price;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void decresePriceByAmount(int amount) {
+        this.price -= amount;
+    }
+
+    public void decresePriceByPercent(int percent) {
+        this.price =  this.price / 100 * (100-percent);
```

Comment:
별거 아닐순 있지만,

```suggestion
        this.price =  this.price / 100 * (100 - percent);
```

코드스타일 맞추는것이 생각보다 많이 중요합니다.